src/database/vehicleDb.ts

```diff
@@ -0,0 +1,23 @@
+import { dbConnection } from './dbConnection';
+import { Vehicle } from '../interfaces/Vehicle';
+
+export const vehicleDb = {
+  async get(vrm: string): Promise<Vehicle> {
+    const connection = await dbConnection();
+
+    const getResult: Vehicle[] = await connection
```

Comment:
You are initializing a variable to verify and return later on. It is not a method or a getter so just `results` would be more adequate.

Reply:
Will amend to results.